### DISCUSSION TITLE
Lambda Perf Benchmark

### DISCUSSION THREAD

**Discussion by RichiCoder1** [aws/aws-lambda-dotnet#1442] [General]
Howdy! Just wanted to highlight a cool lambda perf benchmark I stumbled across: https://maxday.github.io/lambda-perf/

Made a PR to add dotnet7 aot support and has some very interesting numbers! Fingers crossed somehow can drop both the coldstart and hot duration even further :D

**Comment by normj** [comment]
This is great, thanks for sharing.
